src/Worlds.scss

```diff
@@ -172,7 +172,7 @@ body.contrast-theme {
     }
 
     .Scene__grid-line--Space {
-        stroke: #FFFFFF;
+        stroke: #0078FF;
```

Comment:
As with the drawing colour, if they're the same, I'd argue we should just have a single rule for `.Scene__grid-line`.

Reply:
.Scene__grid-line will only have common properties, and we would specify colours in different worlds with --{WorldName} class name. This may introduce some duplication in the codebase, but would be easier to understand the codebase.